packages/kbn-check-mappings-update-cli/current_mappings.json

```diff
@@ -880,6 +880,9 @@
     "properties": {
       "flapping": {
         "properties": {}
+      },
+      "queryDelay": {
+        "properties": {}
```

Comment:
Just FMI, is the intention that this contain searchable fields in the future? I'm not sure this will be useable for any searches at the moment so not sure it is worth adding it to your mappings.

Reply:
I can remove it, thanks!

Reply:
Resolved in this commit https://github.com/elastic/kibana/pull/167433/commits/4c0a8b9c15c2bb68ca57ac542b40142da8fa7385

Reply:
Oh wait, the auto commit added it back

Reply:
Sorry about that, should be resolved in this commit fab99e157c50e302ee49711086ceb4e2d2eeabda